### ISSUE TITLE
Confirmation of deleting a scenario

### ISSUE THREAD

**Issue by chrismaliszewski** [NPBruce/valkyrie#1081]
I think deleting scenarios, no matter of a kind, should be possible from a splash screen so there is no need going back to Editor menu. 

I like the first idea of the dialog. It could be a simple thing:
1. I personally think the Delete button should stand out more and should be red-coloured. Moreover, all Back buttons in the app should have a neutral colour, i.e. be grey as the Delete is now or white. They don't do anything major/serious/dangerous to be red. But it's a topic for a different discussion.
![image](https://user-images.githubusercontent.com/7532602/53131229-0df35a80-35d1-11e9-99d2-bebdc9da839b.png)
2. Clicking the Delete button causes appearing of the confirmation text and two new buttons:
![image](https://user-images.githubusercontent.com/7532602/53131269-219ec100-35d1-11e9-947b-33207bc0a41d.png)
PSD: [delete_confirmation.zip](https://github.com/NPBruce/valkyrie/files/2886866/delete_confirmation.zip)

The design is plain and simple. **It'd could be an enhancement for another release.** 

PS. Grey vs White Back button:
![image](https://user-images.githubusercontent.com/7532602/53131706-5f501980-35d2-11e9-9765-0d0e40360e24.png)

_Originally posted by @chrismaliszewski in https://github.com/NPBruce/valkyrie/issues/1077#issuecomment-465792756_

**Comment by BenRQ** [comment]
I thought about this, and I really think that editor scenario should only be deletable from the editor.

**Comment by chrismaliszewski** [comment]
Why do you think so? Is that because of the safety reasons?

1. Locally created scenarios are only accessible from Offline mode. 
2. There would be the confirmation before deleting a scenario.

If you're still worried:
1. a new filter could be created to hide locally created scenarios and/or...
2. the delete button for the locally created scenarios could be a bit little different from the normal ones, i.e.
![image](https://user-images.githubusercontent.com/7532602/53160315-905d3800-362c-11e9-90cf-591fc15b8668.png)

As I said before, I think deleting scenarios, no matter of a kind, should be possible from the same place (+from Editor if you still want to use it) so there is no need going back to Editor menu. But I think others should say what they think. @scrubbless @redwolf2?

**Comment by BenRQ** [comment]
Yes for me the risk of error is too high compare to the benefit of being able to delete Editor scenario in scenario explorer.

When you create a scenario in the editor, you can only delete it in the editor.
It makes sense to me.

**Comment by chrismaliszewski** [comment]
Point taken. I am not a creator so if you think it's better that way, it's ok. In https://github.com/NPBruce/valkyrie/issues/1077#issuecomment-465809065 you said you've already deleted the possibility. And the only scenarios that will have Delete button will be downloaded ones. 

But in both cases, deleting downloaded and created locally, it'd be good to have a confirmation. I know that deleting scenarios in Editor is a two-step process. But it's better to be sure you don't delete something you don't want to, right? Or think differently?

**Comment by antontimmermans** [comment]
A delete confirmation wouldn't harm anyone and it protects the trigger-happy-clicker against him/her-self. :). 
I agree that scenario in the editor should be only deleted from the scenario editor.

_PS: I have to admit that for me clicking the yes/no for confirmation is more a reflex then a conscious action: are you sure 'yes'... are you really sure "YES" ... really really sure? **Y E S !**_  ;)

**Comment by scrubbless** [comment]
I like the delete confirmation, great idea. But agree with BenRQ, local editor content should only be deletable from editor menu. In my eyes the two content types should be kept independent.

**Comment by redwolf2** [comment]
I see two options:

* Have a message, that explains, that it should be deleted from the editor menu.
* Avoid trigger happyness by having an additional text field, where the user is told, that this is an editor scenario and he has to type in a confirmation text to delete it. Much like the deletion of git repositories work.
